### PR TITLE
Fixes #10720: Made scrollbar optional on wiki pages

### LINKED ISSUE #10720: Make scrollbars optional, or add custom scrollbars
### Please describe the problem (or idea)
In the wiki pages, the tables have scrollbars attached at both horizontal and vertical axes, even though they perform no function in this case. 
image

> What happened just before the problem occurred? Or what problem could this idea solve?
By making the scrollbars either optional based on the content, or making some sort of custom scrollbar, we can make the UI look better.

> What did you expect to see that you didn't?

A non functioning scrollbar.

### Please show us where to look

https://publiclab.org/wiki/micro

anirudhprabhakaran3

Google Chrome - 98.0.4758.82 (Official Build) (64-bit) (cohort: Stable) 
OS - Windows 11 Version 21H2 (Build 22000.493)

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by KarishmaVanwari**
Fixes #10720 

Screenshot:
image

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

File: app/views/grids/_wikis.html.erb [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
-  th {
-    position: sticky;
-    top: 0;
-    z-index: 5;
-    background: #fff;
-}
-</style>
-</div>
+<div class="inline-wiki-container">
+<div style="max-height:500px;overflow:auto;">
```
Comment (anirudhprabhakaran3) [inline app/views/grids/_wikis.html.erb:2]: I believe the default value is `overflow:auto`. Could you try that out once? If that is the case, then maybe we can remove this part of the inline CSS entirely.

Comment (KarishmaVanwari) [reply app/views/grids/_wikis.html.erb:2]: @anirudhprabhakaran3 The default value for overflow is `visible`.
I tried removing the overflow property entirely, so there does not appear any scrollbar even on smaller screen sizes :

image


And with `overflow:auto`, on smaller screen sizes, a scrollbar appears as and when needed:

image


Which one looks better? Or maybe, I could also add a custom scrollbar whenever it appears (i.e. with `overflow:auto`)?

Comment (anirudhprabhakaran3) [reply app/views/grids/_wikis.html.erb:2]: Aah i see that is my mistake. Yeah, overflow:auto looks good and I think that's what we were trying to do here. Looks good!

**Review by anirudhprabhakaran3 — APPROVED** [review]

**Comment by cesswairimu** [comment]
Hi @KarishmaVanwari, would you like to push your branch over to unstable [here](https://unstable.publiclab.org/) we see how the page looks? 

on your branch you could use this command `git push -f https://github.com/publiclab/plots2 HEAD:unstable` _might not work if you are not added as a plots2 contributor_ but you could try and let us know how it goes. Thanks

File: app/views/grids/_wikis.html.erb (hunk 1 above)
Comment (KarishmaVanwari) [reply app/views/grids/_wikis.html.erb:2]: Thanks :)

**Comment by KarishmaVanwari** [comment]
@cesswairimu I am unable to push to unstable using the command. It gives me the following error on the terminal:
`remote: Permission to publiclab/plots2.git denied to KarishmaVanwari.
fatal: unable to access 'https://github.com/publiclab/plots2.git/': The requested URL returned error: 403`

**Comment by cesswairimu** [comment]
okay noted, I will try to add you as a contributor or push your branch, whichever I can and then will revert. Thanks

**Comment by KarishmaVanwari** [comment]
Sure, thanks :) @cesswairimu

**Comment by cesswairimu** [comment]
I couldn't add you as a contributor, but I have pushed... the changes should be available at https://unstable.publiclab.org/ in 10 minutes or so, lets test there. thanks

**Comment by KarishmaVanwari** [comment]
@cesswairimu There isn't any table in any of the wiki pages at https://unstable.publiclab.org/wiki, so how do we check the changes?

**Comment by cesswairimu** [comment]
@KarishmaVanwari with unstable you can find most pages on the live website. For this, try https://unstable.publiclab.org/wiki/micro please

**Comment by KarishmaVanwari** [comment]
@cesswairimu 
I am unable to find the expected changes on https://unstable.publiclab.org/wiki/micro. I figured that the change is to be done for the div with parent class `inline-grid-container`, so I've added a commit for the same.
I doubt if there are any other such div's that need such a change, could you please help me figure that out?

**Comment by KarishmaVanwari** [comment]
@cesswairimu It seems like the div with parent class `inline-grid-container` requires the changes for optional scrollbar. Please do review the changes.

**Comment by cesswairimu** [comment]
Hi @KarishmaVanwari I agree, this is looking good. Thanks

**Review by cesswairimu — APPROVED** [review]
:tada:

**Comment by cesswairimu** [comment]
merging, thanks @KarishmaVanwari for working on this and thanks @anirudhprabhakaran3  for the reviews

**Comment by KarishmaVanwari** [comment]
Thank you @cesswairimu @anirudhprabhakaran3!!

**Comment by anirudhprabhakaran3** [comment]
Great work!

**Pull request merged**